metadata-models/src/main/pegasus/com/linkedin/glossary/GlossaryNodeInfo.pdl

```diff
@@ -0,0 +1,15 @@
+namespace com.linkedin.glossary
+
+import com.linkedin.common.GlossaryNodeUrn
+
+/**
+ * Properties associated with a GlossaryNode
+ */
+record GlossaryNodeInfo {
+
+  /**
+   * Definition of business node
+   */
+  definition: string
```

Comment:
Can we add a reference to the GlossaryNode's parent node here? This way we would be able to access the parent node without having to rely on string parsing of the Node's Urn.

Reply:
storing whole object of parent glossary node inside child could take more space as we can have n number of hierarchy.
Either we can just store urn of the parent GlossaryNode or that information is also available in the urn

Reply:
added parentNode